# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/PBackgroundIDBCursor.ipdl
# Commit: 1f31c24e2ca0
# Full Hash: 1f31c24e2ca0bf2960c98de55396072ce6c46323
# Author: Birunthan Mohanathas <birunthan@mohanathas.com>
# Date: 2015-08-23 21:18:49
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Part 3: Allow multiple ObjectStoreCursorResponses in a CursorResponse. r=khuey
#   
#   For now, we assume that the array only ever contains a single response.
# ==============================================================================

diff -r 756c006944ab -r 1f31c24e2ca0 dom/indexedDB/PBackgroundIDBCursor.ipdl
--- a/dom/indexedDB/PBackgroundIDBCursor.ipdl	Sat Aug 22 10:34:51 2015 +0900
+++ b/dom/indexedDB/PBackgroundIDBCursor.ipdl	Wed Aug 19 14:59:25 2015 -0700
@@ -64,7 +64,7 @@
 {
   void_t;
   nsresult;
-  ObjectStoreCursorResponse;
+  ObjectStoreCursorResponse[];
   ObjectStoreKeyCursorResponse;
   IndexCursorResponse;
   IndexKeyCursorResponse;
